content/news/known-amendments.md

```diff
@@ -9,9 +9,9 @@ The following is a comprehensive list of all known amendments and their status o
 | [OwnerPaysFee][]            | TBD        | [In Development: TBD]( "BADGE_LIGHTGREY") |
 | [SHAMapV2][]                | TBD        | [In Development: TBD]( "BADGE_LIGHTGREY") |
 | [Tickets][]                 | TBD        | [In Development: TBD]( "BADGE_LIGHTGREY") |
-| [fix1578][]                 | v1.2.0     | [Planned: TBD]( "BADGE_LIGHTGREY") |
-| [fixTakerDryOfferRemoval][] | v1.2.0     | [Planned: TBD]( "BADGE_LIGHTGREY") |
-| [MultiSignReserve][]        | v1.2.0     | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [fix1578][]                 | v1.2.0     | [Open for Voting: 2019-02-13](https://ripple.com/dev-blog/introducing-xrp-ledger-rippled-version-1-2-0/ "BADGE_1DB4FF") |
+| [fixTakerDryOfferRemoval][] | v1.2.0     | [Open for Voting: 2019-02-13](https://ripple.com/dev-blog/introducing-xrp-ledger-rippled-version-1-2-0/ "BADGE_1DB4FF") |
+| [MultiSignReserve][]        | v1.2.0     | [Open for Voting: 2019-02-13](https://ripple.com/dev-blog/introducing-xrp-ledger-rippled-version-1-2-0/ "BADGE_1DB4FF") |
```

Comment:
The date here is the date that the release containing these came out. "BADGE_1DB4FF" is light blue (#1DB4FF) because "LIGHTBLUE" is not defined for shields.io.